R/usethis-package.R

```diff
@@ -10,6 +10,68 @@
 ## usethis namespace: end
 NULL
 
+#' Options consulted by usethis
+#'
+#' @description
+#' User-configurable options consulted by usethis, which provide a mechanism
+#' for setting default behaviours for various functions.
+#'
+#' If the built-in defaults don't suit you, set one or more of these options.
+#' Typically, this is done in the `.Rprofile` startup file, which you can open
+#' for editing with [edit_r_profile()] - this will set the specified options for
+#' all future R sessions. Your code will look something like:
+#'
+#' ```
+#' options(
+#'   usethis.description = list(
+#'     `Authors@R` = 'person("Jane", "Doe", email = "jane@example.com",
+#'       role = c("aut", "cre"), comment = c(ORCID = "YOUR-ORCID-ID"))',
```

Comment:
You may be copying this from somewhere else, in which this comment applies to that too. Are we sure this line break is OK, i.e. it doesn't hurt anything?

In my `.Rprofile` I do it like so and I seem to remember that line length was part of the motivation:

```
options(
  ...,
  usethis.description = list(
    "Authors@R" = utils::person(
      "Jennifer", "Bryan",
      email = "jenny@posit.co",
      role = c("aut", "cre"),
      comment = c(ORCID = "0000-0002-6983-2759")
     ),
     License = "MIT + file LICENSE"
    ),
  ...
)
```

Reply:
Yeah, I blindly copied it directly from the `use_description()` docs - I'll fix in both places.

Reply:
This definitely looks nicer

Reply:
Oh actually it's similar in R Packages too: https://r-pkgs.org/setup.html#personal-startup-configuration